diff --git a/base/BUILD.gn b/base/BUILD.gn
index 98bacb75d01cd1ed2da5b88998e4872494f9d0bf..f4eed9bc3923a863c50e5ba9e01e67f0ccb77083 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4380,6 +4380,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
