diff --git a/base/BUILD.gn b/base/BUILD.gn
index aa7f2d6b3f2ff416ea42e320c6813c45ab38049d..5c0a970597c98e1e2956f060c1026356ec208ed4 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4376,6 +4376,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
