diff --git a/base/BUILD.gn b/base/BUILD.gn
index eb9bc951061a48d25e4b87c8ba2c1740163f2a6c..f1507746cd958060c279829cd9d8cd401d539f46 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4377,6 +4377,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
