diff --git a/base/BUILD.gn b/base/BUILD.gn
index 8addf14d0aefcaaa125b1e72b93759375999bcb7..462e47362f7e64d98fb8fdaaa50a9f571db9fe21 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4391,6 +4391,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
