diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
index 0fdb35d316f1cead6c588a62d60345bf50b72405..80268804bbea1ff0dab4cf00f0086b610c4399f1 100644
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -423,6 +423,7 @@ if (enable_java_templates && is_android) {
   #     ]
   #   }
   template("java_cpp_enum") {
+    foreach(source, invoker.sources) { foreach(filter, brave_java_cpp_enum_filter) { if(source == filter) { invoker.sources -= [filter] }}}
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "sources" ])
 
@@ -598,6 +599,7 @@ if (enable_java_templates && is_android) {
   #     output = "$target_gen_dir/AndroidManifest.xml"
   #   }
   template("jinja_template") {
+    if (defined(invoker.includes)) { invoker.includes += brave_android_manifest_includes } else { invoker.includes = brave_android_manifest_includes }
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "deps" ])
       inputs = [ invoker.input ]
