diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index 20a2453085c6fbb7554f972f2ff002173a1746d7..ad5aa9e0ef74504629955bcfb237d493e81d9186 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -22,6 +22,7 @@ declare_args() {
 # toolchain, this GNI file may be read and processed from within Chrome OS
 # toolchains. Checking |is_android| here would therefore be too restrictive.
 if (is_android || is_chromeos) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/chromecast_build.gni")
   import("//build/config/clang/clang.gni")
