diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
index f388c8b42bad1079217d03a58c8232fbe40c3a1f..64067758ede259ea6b52e272c9747d2af2134458 100644
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -213,6 +213,7 @@ if (enable_java_templates && is_android) {
   #     ]
   #   }
   template("java_cpp_enum") {
+    foreach(source, invoker.sources) { foreach(filter, brave_java_cpp_enum_filter) { if(source == filter) { invoker.sources -= [filter] }}}
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "sources" ])
 
@@ -388,6 +389,7 @@ if (enable_java_templates && is_android) {
   #     output = "$target_gen_dir/AndroidManifest.xml"
   #   }
   template("jinja_template") {
+    if (defined(invoker.includes)) { invoker.includes += brave_android_manifest_includes } else { invoker.includes = brave_android_manifest_includes }
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "deps" ])
       inputs = [ invoker.input ]
