diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index 7d9423954b833fe8fedb45acdae2a5c7e957de83..9e15727dc8a7e3d2399aafc0e81f7fb9a27e5731 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -21,6 +21,7 @@ declare_args() {
 # toolchain, this GNI file may be read and processed from within Chrome OS
 # toolchains. Checking |is_android| here would therefore be too restrictive.
 if (is_android || is_chromeos) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/chromecast_build.gni")
   import("//build/config/clang/clang.gni")
