diff --git a/build/toolchain/gcc_toolchain.gni b/build/toolchain/gcc_toolchain.gni
index d31f569d492284646270bfb01fdbfdac950a3dfd..d5ee5de276035a4f95d6eaf6403d17989abb915d 100644
--- a/build/toolchain/gcc_toolchain.gni
+++ b/build/toolchain/gcc_toolchain.gni
@@ -857,6 +857,7 @@ template("gcc_toolchain") {
         is_official_build = false
         use_clang_coverage = false
         use_sanitizer_coverage = false
+        generate_linker_map = false
       }
 
       # When cross-compiling we don't want to use the target platform's file
