diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index ae742b0b85a506f550852e611471338bde0da390..833409b52c694350f05605edcc4fda864b954e43 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1785,6 +1785,7 @@ config("default_warnings") {
   # an error by default eventually; see
   # https://github.com/rust-lang/rust/issues/71668
   rustflags = [ "-Dunsafe_op_in_unsafe_fn" ]
+  rustflags -= brave_remove_rustflags
 }
 
 # prevent_unsafe_narrowing ----------------------------------------------------
@@ -2570,7 +2571,7 @@ config("symbols") {
       if (use_dwarf5) {
         cflags += [ "-gdwarf-5" ]
         rustflags += [ "-Zdwarf-version=5" ]
-      } else if (!is_apple) {
+      } else {
         # Recent clang versions default to DWARF5 on Linux, and Android is about
         # to switch. TODO: Adopt that in controlled way.
         # Apple platforms still default to 4, so the flag is not needed there.
