diff --git a/build/toolchain/apple/toolchain.gni b/build/toolchain/apple/toolchain.gni
index cb29afcd0c7bf953558abb83e726c3009236252e..99933a13c29813928d5192d2359504ebdf85bac4 100644
--- a/build/toolchain/apple/toolchain.gni
+++ b/build/toolchain/apple/toolchain.gni
@@ -840,6 +840,7 @@ template("apple_toolchain") {
         is_official_build = false
         use_clang_coverage = false
         use_sanitizer_coverage = false
+        generate_linker_map = false
       }
     }
   }
