diff --git a/build/toolchain/win/toolchain.gni b/build/toolchain/win/toolchain.gni
index a2a710869feaefd941bcfb1edbf27627bf0ba5fc..19c0222704c1e0df5f8b3d4f033eb6f7addd9495 100644
--- a/build/toolchain/win/toolchain.gni
+++ b/build/toolchain/win/toolchain.gni
@@ -272,7 +272,7 @@ template("single_msvc_toolchain") {
 
       if (toolchain_args.current_cpu == "arm64") {
         if (toolchain_is_clang) {
-          ml = "${cl_prefix}${_clang_bin_path}/clang-cl${_exe} --target=arm64-windows"
+          ml = "${cl_prefix}${_clang_bin_path}/clang-cl${_exe} --target=aarch64-pc-windows"
           if (host_os == "win") {
             # Flip the slashes so that copy/paste of the command works.
             ml = string_replace(ml, "/", "\\")
@@ -643,6 +643,7 @@ template("msvc_toolchain") {
         is_official_build = false
         use_clang_coverage = false
         use_sanitizer_coverage = false
+        generate_linker_map = false
 
         # On Windows, cross-compile for x86 changes the `host_toolchain`
         # into x86 too so as to avoid compiling things twice (see
