diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 157ee5cadc0a7bf97be0fc8bf34adb858beea1f1..a6d84e7eff996e61443a6439c433754e59702575 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2686,6 +2686,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
