diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index 0d934929aa46f79ad5462690bedaddc2e62eb4f7..a4b5335a3f61220c03ce5858e4703d32b4672da8 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -422,6 +422,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
       "safetyCheckShowReviewPanel",
       base::FeatureList::IsEnabled(features::kSafetyCheckExtensions));
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
