diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index c943045f15e30c597b0ae60d17e361cf15a85716..f2b2ebf0b45bb947bd0b99cb664e15018a3fd704 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3809,6 +3809,7 @@ const FeatureEntry::FeatureVariation kPasswordGenerationExperimentVariations[] =
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
