diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index 4a3df6ee2764c915524ded92d579a71678528c96..de122d65dcbac6c9d8874de27115f0d9f5527660 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -225,6 +225,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
