diff --git a/chrome/browser/themes/theme_service_factory.cc b/chrome/browser/themes/theme_service_factory.cc
index f4d6b0df004b61e783942a89d55010ce28d5dced..eac5455cf2a7fa74c205f6ac79bccc600091ca45 100644
--- a/chrome/browser/themes/theme_service_factory.cc
+++ b/chrome/browser/themes/theme_service_factory.cc
@@ -103,6 +103,7 @@ ThemeServiceFactory::~ThemeServiceFactory() = default;
 
 KeyedService* ThemeServiceFactory::BuildServiceInstanceFor(
     content::BrowserContext* profile) const {
+  BRAVE_THEMESERVICEFACTORY_BUILDSERVICEINSTANCEFOR
 #if BUILDFLAG(IS_LINUX)
   using ThemeService = ThemeServiceAuraLinux;
 #endif
