diff --git a/chrome/browser/ui/views/tabs/new_tab_button.h b/chrome/browser/ui/views/tabs/new_tab_button.h
index 9b626289bf0e6c0039d846cea80b37332aff58b6..022b602e8f37c1ea1cc228d17916acc84d970711 100644
--- a/chrome/browser/ui/views/tabs/new_tab_button.h
+++ b/chrome/browser/ui/views/tabs/new_tab_button.h
@@ -50,6 +50,7 @@ class NewTabButton : public views::ImageButton,
   void AddLayerToRegion(ui::Layer* new_layer,
                         views::LayerRegion region) override;
   void RemoveLayerFromRegions(ui::Layer* old_layer) override;
+  BRAVE_NEW_TAB_BUTTON_H_
 
  protected:
   virtual void PaintIcon(gfx::Canvas* canvas);
