diff --git a/chrome/browser/resources/settings/settings.ts b/chrome/browser/resources/settings/settings.ts
index 9f31e56b5156a84407f6d0dd1d3973d67e4ef228..61fd52101d2eed504cee03513e25cdb6eab042af 100644
--- a/chrome/browser/resources/settings/settings.ts
+++ b/chrome/browser/resources/settings/settings.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './settings_ui/settings_ui.js';
 
 export {ControlledRadioButtonElement} from '/shared/settings/controls/controlled_radio_button.js';
