diff --git a/chrome/browser/ui/views/page_info/page_info_main_view.cc b/chrome/browser/ui/views/page_info/page_info_main_view.cc
index bc22c810d289cab47a3214de5619bd3f7852eb07..734bc694ae6c0a75ec6e0246c5cd24c233668bd5 100644
--- a/chrome/browser/ui/views/page_info/page_info_main_view.cc
+++ b/chrome/browser/ui/views/page_info/page_info_main_view.cc
@@ -555,6 +555,7 @@ gfx::Size PageInfoMainView::CalculatePreferredSize() const {
   if (site_settings_view_) {
     width = std::max(width, site_settings_view_->GetPreferredSize().width());
     width = std::max(width, permissions_view_->GetPreferredSize().width());
+    BRAVE_PAGE_INFO_MAIN_VIEW_CALCULATE_PREFERRED_SIZE
   }
   return gfx::Size(width, views::View::GetHeightForWidth(width));
 }
