diff --git a/chrome/browser/ui/views/page_info/page_info_view_factory.cc b/chrome/browser/ui/views/page_info/page_info_view_factory.cc
index 9f7a0b0077b1b90aeccc9e8b0b4b97dd924f4e25..28c872016def1f8babf0b4a7f692e553cb251a8c 100644
--- a/chrome/browser/ui/views/page_info/page_info_view_factory.cc
+++ b/chrome/browser/ui/views/page_info/page_info_view_factory.cc
@@ -405,6 +405,7 @@ const ui::ImageModel PageInfoViewFactory::GetPermissionIcon(
 
   const gfx::VectorIcon* icon = &gfx::kNoneIcon;
   switch (info.type) {
+    BRAVE_PAGE_INFO_VIEW_FACTORY_GET_PERMISSION_ICON
     case ContentSettingsType::COOKIES:
       icon = &vector_icons::kCookieIcon;
       break;
