diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 78e1b28d82e2954b7ff83cd45a42e6c501496974..073079ff4e0f99210988741abe2e3f2ff414b5f2 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2562,6 +2562,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
