diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index 15405463ca735095d035dea1131ee535d5b5c0a2..bb45ef713a30c42a823b32080e4f67dd5afe7504 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -224,6 +224,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
