diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index f48a593052ca8da6353ba47ec77598a5bd0cbd4a..8c295004d5a1c2741d4d3aaa699176a44ef4e753 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -429,6 +429,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
       "safetyCheckShowReviewPanel",
       base::FeatureList::IsEnabled(features::kSafetyCheckExtensions));
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
