diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
index 28da45e0f812ea7b2eed00e8ff4fc765990f2367..1e417ba9e4b41fe70a46d021e51ee28fc6aa7c30 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
@@ -82,6 +82,7 @@ class RecentTabsSubMenuModel : public ui::SimpleMenuModel,
 
   void RegisterLogMenuMetricsCallback(LogMenuMetricsCallback callback);
 
+  BRAVE_RECENT_TABS_SUB_MENU_MODEL_H_
  private:
   struct TabNavigationItem;
   using TabNavigationItems = std::map<int, TabNavigationItem>;
