diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 374e9dfdf8dfe5c6aeabfd185810c656bb92cd21..db004b3add001074d1622ba88543e72576db2fbf 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -414,6 +414,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data_deps = []
 
   if (enable_pdf) {
