diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
index 573d184a7246b5c4ed4842ead4805abdc60003b8..c3f4c36958e717d941062717fd231f121ca1f741 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
@@ -82,6 +82,7 @@ class RecentTabsSubMenuModel : public ui::SimpleMenuModel,
 
   void RegisterLogMenuMetricsCallback(LogMenuMetricsCallback callback);
 
+  BRAVE_RECENT_TABS_SUB_MENU_MODEL_H_
  private:
   struct TabNavigationItem;
   using TabNavigationItems = std::map<int, TabNavigationItem>;
