diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc
index 9bbf28cbd711e1c738fc473cb878a440fa31b9d4..0b16f1c5adf3cb0a08c1103d03b16d7e90f2fa02 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc
@@ -471,6 +471,7 @@ void RecentTabsSubMenuModel::BuildTabsFromOtherDevices() {
            ++k) {
         BuildOtherDevicesTabItem(this, session_tag, *tabs_in_session[k]);
       }  // for all tabs in one session
+      BRAVE_BUILD_TABS_FROM_OTHER_DEVICES
     }
     ++num_sessions_added;
   }  // for all sessions
