diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc
index ff5577127931cea770e3633e52e9f4e6a87da688..5527e718a954a672a51b5e10b3794b7a8bc2edbe 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc
@@ -442,6 +442,8 @@ void RecentTabsSubMenuModel::BuildTabsFromOtherDevices() {
          k < std::min(tabs_in_session.size(), kMaxTabsPerSessionToShow); ++k) {
       BuildOtherDevicesTabItem(session_tag, *tabs_in_session[k]);
     }  // for all tabs in one session
+
+    BRAVE_BUILD_TABS_FROM_OTHER_DEVICES
     ++num_sessions_added;
   }  // for all sessions
 
