diff --git a/chrome/browser/resources/settings/people_page/sync_controls.ts b/chrome/browser/resources/settings/people_page/sync_controls.ts
index 400a2cc2245a3f5ac07000356eab9aab65f76bca..80e95d78f00b61414a6707eb9f0097f9653fb638 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.ts
+++ b/chrome/browser/resources/settings/people_page/sync_controls.ts
@@ -118,7 +118,7 @@ export class SettingsSyncControlsElement extends
 
     const router = Router.getInstance();
     if (router.getCurrentRoute() ===
-        (router.getRoutes() as {SYNC_ADVANCED: Route}).SYNC_ADVANCED) {
+        (router.getRoutes() as {BRAVE_SYNC_SETUP: Route}).BRAVE_SYNC_SETUP) {
       this.browserProxy_.didNavigateToSyncPage();
     }
   }
