diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index d6d2513ed2093501a027cb54102828332fa5499f..900fd196d01d6ee3d0b2597a66144c10263a2643 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4039,6 +4039,7 @@ const FeatureEntry::FeatureVariation kAndroidHubVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
