diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index 84006f8966e17e572e9b4a8324d6e43744b87ee6..761b644481469bda2f4aca0db175b06dc593e8e6 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -430,6 +430,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
       "safetyCheckShowReviewPanel",
       base::FeatureList::IsEnabled(features::kSafetyCheckExtensions));
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
