diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model_unittest.cc b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model_unittest.cc
index 093107a7e9b6348dff37e005f4ef4bd331b21e04..449d30cbfd4cc0944e2991fa1af73756a1d3a8c2 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model_unittest.cc
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model_unittest.cc
@@ -166,6 +166,7 @@ class RecentTabsSubMenuModelTest : public BrowserWithTestWindowTest {
     helper->VerifyExport(session_sync_service_->GetOpenTabsUIDelegate());
   }
 
+  BRAVE_RECENT_TABS_SUB_MENU_MODEL_TEST
  private:
   raw_ptr<sync_sessions::SessionSyncService, DanglingUntriaged>
       session_sync_service_;
@@ -560,8 +561,8 @@ TEST_F(RecentTabsSubMenuModelTest, MaxSessionsAndRecency) {
   EXPECT_THAT(base::span<const std::u16string>(
                   recent_tabs_builder.GetTabTitlesSortedByRecency())
                   .subspan(0, 3),
-              ElementsAre(model.GetLabelAt(5), model.GetLabelAt(8),
-                          model.GetLabelAt(11)));
+              ElementsAre(model.GetLabelAt(6), model.GetLabelAt(9),
+                          model.GetLabelAt(12)));
 }
 
 TEST_F(RecentTabsSubMenuModelTest, MaxTabsPerSessionAndRecency) {
@@ -598,6 +599,6 @@ TEST_F(RecentTabsSubMenuModelTest, MaxTabsPerSessionAndRecency) {
   EXPECT_THAT(base::span<const std::u16string>(
                   recent_tabs_builder.GetTabTitlesSortedByRecency())
                   .subspan(0, 4),
-              ElementsAre(model.GetLabelAt(5), model.GetLabelAt(6),
-                          model.GetLabelAt(7), model.GetLabelAt(8)));
+              ElementsAre(model.GetLabelAt(6), model.GetLabelAt(7),
+                          model.GetLabelAt(8), model.GetLabelAt(9)));
 }
