diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 8f717ddb019922921b46eb982680275b8f1a81d2..7a8ece70be6e14459f75472f2e72c93dff56685e 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2567,6 +2567,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
