diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 238590fde59d43e4755bb8d4254bc7ae5f32647a..e0d8a9b80e09446c39d9ce0ca641eb18279f0b64 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1401,6 +1401,7 @@ static_library("extensions") {
   } else {
     sources += [ "api/braille_display_private/braille_controller_stub.cc" ]
   }
+  deps += brave_extensions_resources
 }
 
 proto_library("cws_item_service_proto") {
