diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index e654b3d6c046585a50500efa5b701acbdb03c50f..2f3e5e5d26def71664dd987e28332882cf27b646 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -72,6 +72,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
