diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 375dae6d34363c75f9881cb8a9308494b5a80e6b..fb2be6056be2f1290f355154e65b3cd5b48e3896 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4034,6 +4034,7 @@ const FeatureEntry::FeatureVariation kAndroidHubVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
