diff --git a/chrome/browser/ui/android/toolbar/BUILD.gn b/chrome/browser/ui/android/toolbar/BUILD.gn
index 7a7c3c823bd989140a86d8ce7e9eadda71b527e4..661ef62549c15f2a8923e12b6fbdad1d2dfb1f86 100644
--- a/chrome/browser/ui/android/toolbar/BUILD.gn
+++ b/chrome/browser/ui/android/toolbar/BUILD.gn
@@ -117,6 +117,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/toolbar/top/ViewShiftingActionBarDelegate.java",
     "java/src/org/chromium/chrome/browser/toolbar/top/VisibleUrlText.java",
   ]
+  sources += brave_browser_ui_android_toolbar_sources
 
   deps = [
     ":java_resources",
