diff --git a/chrome/browser/ui/views/tabs/new_tab_button.h b/chrome/browser/ui/views/tabs/new_tab_button.h
index 6f307b2d37eb079b5a563953aaa0271925bfddc9..018e36c7acc36c5152b43881d126a8d46a84d0d9 100644
--- a/chrome/browser/ui/views/tabs/new_tab_button.h
+++ b/chrome/browser/ui/views/tabs/new_tab_button.h
@@ -49,6 +49,7 @@ class NewTabButton : public views::ImageButton,
   void AddLayerToRegion(ui::Layer* new_layer,
                         views::LayerRegion region) override;
   void RemoveLayerFromRegions(ui::Layer* old_layer) override;
+  BRAVE_NEW_TAB_BUTTON_H_
 
  protected:
   virtual void PaintIcon(gfx::Canvas* canvas);
