diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 041ea590b5e9e2e23f9d04e0be19ed7222ecbc41..2736f1b38e9ece408eac8f08740adffa50413a7f 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4011,6 +4011,7 @@ const FeatureEntry::FeatureVariation kWebAuthnAndroidCredManVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
