diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index a85890bc3e169b864fd64ce1e3d49a3c060b7f3c..317bfe908f52fb346573e96c01aca2a15708b81e 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3760,6 +3760,7 @@ const FeatureEntry::FeatureVariation kPasswordGenerationExperimentVariations[] =
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
