diff --git a/components/password_manager/core/browser/login_database.cc b/components/password_manager/core/browser/login_database.cc
index aa0dfeac06949511b9fad707b13c08004156c80b..cd867031f9d35f5e0ab2cbed618d4a8904c86e65 100644
--- a/components/password_manager/core/browser/login_database.cc
+++ b/components/password_manager/core/browser/login_database.cc
@@ -2029,6 +2029,7 @@ FormRetrievalResult LoginDatabase::StatementToForms(
 
     EncryptionResult result = InitPasswordFormFromStatement(
         *statement, /*decrypt_and_fill_password_value=*/true, new_form.get());
+    BRAVE_STATEMENT_TO_FORMS
     if (result == ENCRYPTION_RESULT_SERVICE_FAILURE) {
       has_service_failure = true;
       continue;
