diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index 0dcd3d2b7e2d0691ceb142b2fd24981871dd9153..3ee35cb4070514b5019001b9514262313c1195ab 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -141,6 +141,7 @@ source_set("permissions") {
   if (!is_android && !is_ios) {
     deps += [ "//components/guest_view/browser" ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
