diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index 28c4ff4c535f7ca935a793e785636a4d892be25b..8ae2c91a9db9dc86270d9d13f835b6c1cb5e008d 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -16,6 +16,7 @@ group("sync") {
   if (is_chromeos) {
     public_deps += [ "//components/sync/chromeos" ]
   }
+  import("//brave/build/config/brave_build.gni") public_deps += brave_components_sync_deps
 }
 
 static_library("test_support") {
