diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index fc8fe4fa8e253c389538bfd7f2fecbf16db7d247..3ccb06dd38b4c11b72b9adda866d846f75ec3c78 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -252,6 +252,7 @@ class SyncServiceImpl : public SyncService,
   bool IsSyncFeatureConsideredRequested() const override;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
