diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 18c62895714ae2fc18d79fbcbf3bc31640467cb1..203222e18d8830e880208fe8e9d51cf103ad04ff 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -120,6 +120,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kTopLevelStorageAccess:
       return RequestTypeForUma::PERMISSION_TOP_LEVEL_STORAGE_ACCESS;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
