diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index bc43bdfed45404313834d2de2b8e9ad068c4d520..0fdea09a0088a4a85eb838fcb11949000b8ca514 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -54,6 +54,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
