diff --git a/components/sync/engine/model_type_worker.h b/components/sync/engine/model_type_worker.h
index b26371554ba9375f210bb586953ac56e64583400..ca2b13961b48d244c5b37b4caf42e48fc75b987e 100644
--- a/components/sync/engine/model_type_worker.h
+++ b/components/sync/engine/model_type_worker.h
@@ -207,6 +207,7 @@ class ModelTypeWorker : public UpdateHandler,
 
   static constexpr size_t kMaxPendingInvalidations = 10u;
 
+ BRAVE_MODEL_TYPE_WORKER_H_
  private:
   struct UnknownEncryptionKeyInfo {
     // Not increased if the cryptographer knows it's in a pending state
