diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index ab6b75cb57e15d94e2b9fcd64906f3ef6ccf85ff..0d44b222066cf45f46780e4c1dd2b76c49e6922e 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -289,6 +289,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
