diff --git a/components/sync/service/BUILD.gn b/components/sync/service/BUILD.gn
index f5575747398e3d67dac2f6f09fe54a5b08f8e3bb..8475a4e313d06969e8159e89002c4fb138cb77b5 100644
--- a/components/sync/service/BUILD.gn
+++ b/components/sync/service/BUILD.gn
@@ -113,4 +113,5 @@ static_library("service") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
