diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 8a0af7bd17cefe2847010021afbeed656f1aca11..85af08b5684c6966dd8d56835aa92620208ecd38 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1179,6 +1179,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
