diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index 871b295f96c60e8b4baee3c832d6d1bc33c544de..7c809e269866c74dafaf41ca18aaf7e5471f6064 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -16,6 +16,7 @@ group("sync") {
   if (is_chromeos) {
     public_deps += [ "//components/sync/chromeos" ]
   }
+  import("//brave/build/config/brave_build.gni") public_deps += brave_components_sync_deps
 }
 
 static_library("test_support") {
