diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 5bb0af252f0787dc2aada0eb55d1094c9840e413..dbc6665e4ea1f3632056a7703e0d1a60f49f037d 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -120,6 +120,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kTopLevelStorageAccess:
       return RequestTypeForUma::PERMISSION_TOP_LEVEL_STORAGE_ACCESS;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
