diff --git a/components/content_settings/core/browser/cookie_settings.cc b/components/content_settings/core/browser/cookie_settings.cc
index de6b4a04dbc61be069ab49862d34f7d25b0398db..4dd33f03daba1582b93799b77f8415478cc4c847 100644
--- a/components/content_settings/core/browser/cookie_settings.cc
+++ b/components/content_settings/core/browser/cookie_settings.cc
@@ -237,6 +237,7 @@ ContentSetting CookieSettings::GetCookieSettingInternal(
                      !first_party_url.SchemeIs(extension_scheme_);
 
   bool block = block_third && is_third_party_request;
+  BRAVE_COOKIE_SETTINGS_GET_COOKIES_SETTINGS_INTERNAL
 
   if (!block) {
     FireStorageAccessHistogram(
