diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 931a643339eee85a136dafce38fae4ae7eddb35d..c14640f58aacea416ab2133a3e87740c4f88d62a 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -122,6 +122,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kTopLevelStorageAccess:
       return RequestTypeForUma::PERMISSION_TOP_LEVEL_STORAGE_ACCESS;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
