diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index 6b74adccb4e30b6df576df697b6dae8bbefad9e6..303dc992914feca7f196580355c6608a9a89a18a 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -398,6 +398,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
         return "window_placement";
       }
 #endif
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
