diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 46d3afbd50789fca0da875f2eabe90697d730e36..6a1465eaf673d1714fe801fbbf855a77cd9d9cb0 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -243,6 +243,7 @@ class SyncServiceImpl : public SyncService,
   bool IsSyncFeatureConsideredRequested() const override;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
