diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index d4e57ea8063d118bbc1761b121046af66b17423e..08141c5c1eba4f4fd320ffc43e1a3aa121de756f 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -325,6 +325,7 @@ void PermissionManager::RequestPermissionsFromCurrentDocument(
         permission_status_callback) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
   const GURL requesting_origin =
+      BRAVE_FORCED_REQUESTING_ORIGIN
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   RequestPermissionsInternal(permissions_types, render_frame_host,
                              requesting_origin, user_gesture,
@@ -375,6 +376,7 @@ PermissionManager::GetPermissionResultForCurrentDocument(
       PermissionUtil::PermissionTypeToContentSettingType(permission);
 
   const GURL requesting_origin =
+      BRAVE_FORCED_REQUESTING_ORIGIN
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   const GURL embedding_origin =
       GetEmbeddingOrigin(render_frame_host, requesting_origin);
