diff --git a/components/content_settings/core/browser/content_settings_pref_provider.h b/components/content_settings/core/browser/content_settings_pref_provider.h
index d51f1b40c5f9a46cb778d51e8e602bc48f2845bb..8ce2e5a76c72dc3872a161900d28dc8dc20dee92 100644
--- a/components/content_settings/core/browser/content_settings_pref_provider.h
+++ b/components/content_settings/core/browser/content_settings_pref_provider.h
@@ -72,6 +72,7 @@ class PrefProvider : public UserModifiableProvider {
   ContentSettingsPref* GetPref(ContentSettingsType type) const;
 
  private:
+  friend class BravePrefProvider;
   friend class DeadlockCheckerObserver;  // For testing.
 
   void Notify(const ContentSettingsPattern& primary_pattern,
