diff --git a/components/sync/service/BUILD.gn b/components/sync/service/BUILD.gn
index 4202386984c1f8d7c37318dc2f11111ba7656787..62a71c107cbdec06f7ccf1ddd04b252b15d2e9c2 100644
--- a/components/sync/service/BUILD.gn
+++ b/components/sync/service/BUILD.gn
@@ -110,4 +110,5 @@ static_library("service") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
