diff --git a/components/content_settings/core/common/content_settings.cc b/components/content_settings/core/common/content_settings.cc
index ba9e88292037309c9c5a265cebc3b46881755491..fb64fb4575e71b86bef18336d35288eb0ee63d43 100644
--- a/components/content_settings/core/common/content_settings.cc
+++ b/components/content_settings/core/common/content_settings.cc
@@ -128,6 +128,7 @@ constexpr auto kHistogramValue = base::MakeFixedFlatMap<ContentSettingsType,
 
     // As mentioned at the top, please don't forget to update ContentType in
     // enums.xml when you add entries here!
+    BRAVE_HISTOGRAM_VALUE_LIST
 });
 
 constexpr int kkHistogramValueMax = std::max_element(
