diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index b1f97b5e83e9ab615b64c025a2ebf32b628673ea..50ce986171d5abcb14b0d63bc1f31e71afddfff0 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -287,6 +287,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
