diff --git a/components/bookmarks/browser/bookmark_model.h b/components/bookmarks/browser/bookmark_model.h
index 0d182b3c87ff9036cf56aebd47aa030a2a35e48a..45a13cd7035b096ff461886ec2042837d5ccb28b 100644
--- a/components/bookmarks/browser/bookmark_model.h
+++ b/components/bookmarks/browser/bookmark_model.h
@@ -385,6 +385,7 @@ class BookmarkModel final : public BookmarkUndoProvider,
   // this is a no-op. Deletion is done asynchronously on a background thread.
   static void WipeAccountStorageForRollback(const base::FilePath& profile_path);
 
+  BRAVE_BOOKMARK_MODEL_H
  private:
   friend class BookmarkCodecTest;
   friend class BookmarkModelFaviconTest;
