diff --git a/components/sync/service/BUILD.gn b/components/sync/service/BUILD.gn
index f4b81400e566e30afe5124bbfea0604e8ec9391b..c5e05ada22b09fdc7939fdb448320a9692b10b74 100644
--- a/components/sync/service/BUILD.gn
+++ b/components/sync/service/BUILD.gn
@@ -112,4 +112,5 @@ static_library("service") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
