diff --git a/components/bookmarks/browser/bookmark_model.h b/components/bookmarks/browser/bookmark_model.h
index 183c0330ce9f778f37db1ab02172e9d65b45cbf8..8d9fe27e148ac66701e0a4928bb6cc1de1441b52 100644
--- a/components/bookmarks/browser/bookmark_model.h
+++ b/components/bookmarks/browser/bookmark_model.h
@@ -385,6 +385,7 @@ class BookmarkModel final : public BookmarkUndoProvider,
   // this is a no-op. Deletion is done asynchronously on a background thread.
   static void WipeAccountStorageForRollback(const base::FilePath& profile_path);
 
+  BRAVE_BOOKMARK_MODEL_H
  private:
   friend class BookmarkCodecTest;
   friend class BookmarkModelFaviconTest;
