diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index 9fa5ba83111927b9c0b95b7a7dee929679f11558..ae23eac7fb3b82452f21fa4f75b9e0c692abd813 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -16,6 +16,7 @@ group("sync") {
   if (is_chromeos) {
     public_deps += [ "//components/sync/chromeos" ]
   }
+  import("//brave/build/config/brave_build.gni") public_deps += brave_components_sync_deps
 }
 
 static_library("test_support") {
