diff --git a/components/sync/engine/model_type_worker.h b/components/sync/engine/model_type_worker.h
index a398bab39420b87f3bda8810202806590bf78fd7..332f98b87462dc12f5f80f74b847565679e98273 100644
--- a/components/sync/engine/model_type_worker.h
+++ b/components/sync/engine/model_type_worker.h
@@ -207,6 +207,7 @@ class ModelTypeWorker : public UpdateHandler,
 
   static constexpr size_t kMaxPendingInvalidations = 10u;
 
+ BRAVE_MODEL_TYPE_WORKER_H_
  private:
   struct UnknownEncryptionKeyInfo {
     // Not increased if the cryptographer knows it's in a pending state
