diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index a8e818aae4d89db9073117311f46b298647cb697..beee9be2561f9447b77ddca3d4d6c6d013885e6c 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -288,6 +288,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
