diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index b6b4048096fe0571d5069eb1637effdc1c67f777..78b04b13d9058fbc0e1ddf587aa978ae7ead7765 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -245,6 +245,7 @@ class SyncServiceImpl : public SyncService,
   bool IsSyncFeatureConsideredRequested() const override;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
