diff --git a/content/browser/resources/gpu/info_view.ts b/content/browser/resources/gpu/info_view.ts
index 9af2ea08c2b805b1e5a5eaac2ecb307d8bea74a7..6155937bd2e61f1afc2c9e63adc4122ce67a085e 100644
--- a/content/browser/resources/gpu/info_view.ts
+++ b/content/browser/resources/gpu/info_view.ts
@@ -538,6 +538,7 @@ export class InfoViewElement extends CustomElement {
           value: clientInfo.graphics_backend,
         },
         {description: 'Command Line', value: clientInfo.command_line},
+        {description: 'Executable path', value: clientInfo.executable_path},
       ]);
     } else {
       sections.clientInfo.list.textContent = '... loading ...';
