diff --git a/content/common/frame.mojom b/content/common/frame.mojom
index b0b0236d58b97bae0c80998c5eff4ede5879de00..b982d0d0634c492ac183e53a9576157b1f92a3c4 100644
--- a/content/common/frame.mojom
+++ b/content/common/frame.mojom
@@ -582,6 +582,7 @@ struct CreateNewWindowParams {
   // Whether the opener will be suppressed in the new window, in which case
   // scripting the new window is not allowed.
   bool opener_suppressed;
+  bool no_javascript_access;
 
   // Whether new window is created for a form submission.
   bool is_form_submission;
