diff --git a/content/common/frame.mojom b/content/common/frame.mojom
index ceff23b4974d0dcc52062a4b3aac170ebd8da385..2432f6f116696bfc8e8baebab83e7d2cd1525965 100644
--- a/content/common/frame.mojom
+++ b/content/common/frame.mojom
@@ -590,6 +590,7 @@ struct CreateNewWindowParams {
   // Whether the opener will be suppressed in the new window, in which case
   // scripting the new window is not allowed.
   bool opener_suppressed;
+  bool no_javascript_access;
 
   // Whether new window is created for a form submission.
   bool is_form_submission;
