diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index 34eeb3f99b4cb44e070f75106bcb21fa7579353b..9f0e3529435b7291b33eede09103bcb009cca715 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -516,6 +516,7 @@ base::Time CanonicalCookie::ValidateAndAdjustExpiryDate(
     // * network_handler.cc::MakeCookieFromProtocolValues
     fixed_creation_date = base::Time::Now();
   }
+  BRAVE_CANONICAL_COOKIE_VALIDATE_AND_ADJUST_EXPIRY_DATE
   if (base::FeatureList::IsEnabled(features::kClampCookieExpiryTo400Days)) {
     base::Time maximum_expiry_date = fixed_creation_date + base::Days(400);
     if (expiry_date > maximum_expiry_date)
