diff --git a/services/network/cookie_manager.cc b/services/network/cookie_manager.cc
index eb8a44a1e8ef9d7fbd7c7a820e7ec9cd7d6bcae6..e19b64bb9b919734e9aee7d7deccc38af924b56a 100644
--- a/services/network/cookie_manager.cc
+++ b/services/network/cookie_manager.cc
@@ -383,6 +383,7 @@ CookieDeletionInfo DeletionFilterToInfo(mojom::CookieDeletionFilterPtr filter) {
   delete_info.name = std::move(filter->cookie_name);
   delete_info.url = std::move(filter->url);
   delete_info.host = std::move(filter->host_name);
+  BRAVE_DELETIONFILTERTOINFO
 
   switch (filter->session_control) {
     case mojom::CookieDeletionSessionControl::IGNORE_CONTROL:
