diff --git a/services/network/cors/cors_url_loader.cc b/services/network/cors/cors_url_loader.cc
index 4bca517f7d0f686b3f3e95cab2643f5dcb1b33a8..bd8cd1cac4e1ca04f32753f20ab9c97d3bad3e86 100644
--- a/services/network/cors/cors_url_loader.cc
+++ b/services/network/cors/cors_url_loader.cc
@@ -759,6 +759,7 @@ void CorsURLLoader::StartRequest() {
       (fetch_cors_flag_ ||
        (request_.method != net::HttpRequestHeaders::kGetMethod &&
         request_.method != net::HttpRequestHeaders::kHeadMethod))) {
+    BRAVE_CORS_URL_LOADER_START_REQUEST
     if (tainted_) {
       request_.headers.SetHeader(net::HttpRequestHeaders::kOrigin,
                                  url::Origin().Serialize());
