diff --git a/services/network/public/cpp/BUILD.gn b/services/network/public/cpp/BUILD.gn
index c234d4bd248926a75f695d6a4b21049aa2b1d5ec..8002dd124c5bbed28282f532fd8b297af5ee24e6 100644
--- a/services/network/public/cpp/BUILD.gn
+++ b/services/network/public/cpp/BUILD.gn
@@ -189,6 +189,7 @@ component("cpp") {
     "//services/proxy_resolver/public/mojom",
   ]
 
+  public_deps += [ "//brave/services/network/public/cpp" ]
   defines = [ "IS_NETWORK_CPP_IMPL" ]
 }
 
