diff --git a/services/network/cookie_settings.cc b/services/network/cookie_settings.cc
index 593dba2eefc3c05a7098ec9f2c48fa9d22d6e62e..b893e460ca3cc77f8ea8edd8a3b31dfa062d9015 100644
--- a/services/network/cookie_settings.cc
+++ b/services/network/cookie_settings.cc
@@ -216,6 +216,7 @@ CookieSettings::GetCookieSettingWithMetadata(
       storage_access_result =
           net::cookie_util::StorageAccessResult::ACCESS_BLOCKED;
     }
+    BRAVE_COOKIE_SETTINGS_GET_COOKIE_SETTINGS_INTERNAL
   }
 
   if (cookie_setting != CONTENT_SETTING_BLOCK && !found_explicit_setting &&
