diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
index 80a52fff8182872e9f2b75de3baa2d4bb57fb9e9..5069ff03bfe66d35397b778be7df3bf4c9ae7ae1 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
@@ -2040,6 +2040,7 @@ ImageData* BaseRenderingContext2D::getImageData(
 }
 
 ImageData* BaseRenderingContext2D::getImageDataInternal(
+    BRAVE_GET_IMAGE_DATA_PARAMS
     int sx,
     int sy,
     int sw,
@@ -2182,6 +2183,7 @@ ImageData* BaseRenderingContext2D::getImageDataInternal(
       DCHECK(!bounds.intersect(SkIRect::MakeXYWH(sx, sy, sw, sh)));
     }
   }
+  BRAVE_GET_IMAGE_DATA
 
   return image_data;
 }
