diff --git a/third_party/rust/serde/v1/BUILD.gn b/third_party/rust/serde/v1/BUILD.gn
index ca142e41d178a1a78206702a2b8f22c96e6e04ad..27c2c1b7c2aa81fe9d744c05b8f807c39596272e 100644
--- a/third_party/rust/serde/v1/BUILD.gn
+++ b/third_party/rust/serde/v1/BUILD.gn
@@ -56,4 +56,5 @@ cargo_crate("lib") {
   ]
   build_root = "crate/build.rs"
   build_sources = [ "crate/build.rs" ]
+  features += [ "alloc", "rc" ]
 }
