diff --git a/third_party/blink/renderer/core/frame/dom_window.h b/third_party/blink/renderer/core/frame/dom_window.h
index 7aa0a7d069e99fda36dfb5618efae7c78a76e6a4..028b65a11a802a8a2505bad0fb1cec48f8adf0ad 100644
--- a/third_party/blink/renderer/core/frame/dom_window.h
+++ b/third_party/blink/renderer/core/frame/dom_window.h
@@ -190,6 +190,7 @@ class CORE_EXPORT DOMWindow : public EventTarget {
 
   void DisconnectFromFrame() { frame_ = nullptr; }
 
+  BRAVE_DOM_WINDOW_H
  private:
   void DoPostMessage(scoped_refptr<SerializedScriptValue> message,
                      const MessagePortArray&,
