diff --git a/tools/clang/pylib/clang/compile_db.py b/tools/clang/pylib/clang/compile_db.py
index e459d9b98e9ea4570ba599acc6ce3583c9d6cc45..4d28361fd1e8a19a1b729a73efc4713d6883856a 100755
--- a/tools/clang/pylib/clang/compile_db.py
+++ b/tools/clang/pylib/clang/compile_db.py
@@ -211,3 +211,4 @@ def Read(path):
   """
   with open(os.path.join(path, 'compile_commands.json'), 'rb') as db:
     return json.load(db)
+import os, sys; sys.path.append(os.path.join(os.path.dirname(__file__), *[os.pardir] * 4, 'brave', 'script')); from import_inline import inline_file_from_src; inline_file_from_src("brave/chromium_src/tools/clang/pylib/clang/compile_db.py", globals(), locals())
