diff --git a/ui/webui/resources/BUILD.gn b/ui/webui/resources/BUILD.gn
index 6236294a3b973e208ec405cb3bb8cdc8e5f09823..4c22e8ff61c044f5e8fc9bec0eba8094e1ad6382 100644
--- a/ui/webui/resources/BUILD.gn
+++ b/ui/webui/resources/BUILD.gn
@@ -118,6 +118,7 @@ generate_grd("build_grd") {
       "$root_gen_dir/ui/webui/resources/cr_components/page_image_service/resources.grdp",
     ]
   }
+  import("//brave/ui/webui/resources/sources.gni") grdp_files += brave_resources_extra_grdps public_deps += brave_resources_extra_grdps_deps
 }
 
 if (is_chromeos_ash) {
