diff --git a/BUILD.gn b/BUILD.gn
index 2149feddb53a69f8ee15af7acd30b9f24932bc42..fea88338e2653ef6e30122118afabc05c45a4a30 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -728,6 +728,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -2978,6 +2979,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
