diff --git a/BUILD.gn b/BUILD.gn
index 3c028361414720636cdabbc585e4fa508614f661..f65d0a0232076c6054c31bf5eb901d84c7ee6564 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -722,6 +722,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -2962,6 +2963,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
