diff --git a/BUILD.gn b/BUILD.gn
index 5d330cfd213f21c92698156c3155e89d5262e99e..6a8e566410771ee5b9a5e9dc29b8cddc00e0bdbe 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -717,6 +717,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -2949,6 +2950,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
