diff --git a/BUILD.gn b/BUILD.gn
index 8e32175a7301134dee7daaa121f4f53ab2468798..9876c86b9f30a982d1bc738c04a546cc29d5af03 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -730,6 +730,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -2996,6 +2997,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
