diff --git a/BUILD.gn b/BUILD.gn
index d35112c2a6e7ba2b652aec273dab36ca71682c84..c96870900b8e654e26c24a999bf67d14adc0586d 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -730,6 +730,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -2996,6 +2997,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
