diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 7988c0dfd0283381df35701a3ba885f5b70ba6a5..728b3d7ddcc67490af494408fc4f67190de4baf0 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2948,6 +2948,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
