diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 79d0abb3794c57cc8d780ffc98bbb004e873e696..1b2c48307868fd9032eb425ea8ce336a57a09512 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2952,6 +2952,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
